packages/sanity/src/core/studio/StudioErrorBoundary.tsx

```diff
@@ -19,6 +19,12 @@ import {CorsOriginError} from '../store'
 import {isRecord} from '../util'
 import {CorsOriginErrorScreen, SchemaErrorsScreen} from './screens'
 
+const DevServerStoppedErrorScreen = lazy(() =>
+  import('./DevServerStopped').then((DevServerStopped) => ({
```

Comment:
Something that crossed my mind: if we are lazily importing this component only after we detect the dev server has gone away, is there any scenario in which it won't be available at that point in time?

Reply:
Hmm that's interesting. So I don't believe so- we would have to have loaded this module in the first place to throw such an error. I feel it worthy a code comment though as for a few mins I was wondering how this was even working 😖